.github/workflows/ci.yml

```diff
@@ -123,6 +124,38 @@ jobs:
       with:
           command: clippy
           args: --all --tests --benches -- -D warnings
+
+    - name: build release
```

Comment:
you should restrict this to only master not prs I think

Reply:
It is (admittedly very weird as it just filters by branch name 2 lines lower `github.ref_name=='main'`) as I still need to work out the whole thing for other architectures.